docs/examples/model_ex/model_examples_pull_no_model_unc.py

```diff
@@ -58,6 +58,6 @@
     model_uncertainty=False,  # <--
 )
 
-add_luminosity(collaboration="Beast III", ax=ax_main, is_data=False)
+add_luminosity(collaboration="Plothist", ax=ax_main, is_data=False)
```

Comment:
I liked showing a space in the example for the collaboration, as people might do unnecessary `\,`  or `$$` modes.

Reply:
Also do we want upper case `Plohist`? aren't we lowercase `plothist`?

Reply:
I changed to lower case. For the space, not sure it is necessary, as a user, I would naively assume that I do not need \,